bring/src/test/java/com/bobocode/petros/bring/scanner/DefaultBeanPostProcessorScannerTest.java

```diff
@@ -0,0 +1,97 @@
+package com.bobocode.petros.bring.scanner;
+
+import com.bobocode.petros.bring.factory.postprocessor.BeanPostProcessor;
+import com.bobocode.petros.bring.scanner.impl.DefaultBeanPostProcessorScanner;
+import com.bobocode.petros.bring.scanner.mocks.EveningBeanPostProcessor;
+import com.bobocode.petros.bring.scanner.mocks.EveningService;
+import com.bobocode.petros.bring.scanner.mocks.MorningBeanPostProcessor;
+import com.bobocode.petros.bring.scanner.mocks.MorningService;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.MockitoAnnotations.openMocks;
+
+@ExtendWith({MockitoExtension.class})
+class DefaultBeanPostProcessorScannerTest {
+    private final Set<Class<?>> classes = new HashSet<>();
+    private DefaultBeanPostProcessorScanner scanner;
+
+    @BeforeEach
+    void setup() {
+        openMocks(this);
```

Comment:
Do we need this line? I don't see any mocking logic here... The logic is tested using the real classes, so probably it make sense to remove this line.